packages/web-components/src/components/va-radio-option/va-radio-option.tsx

```diff
@@ -104,6 +104,7 @@ export class VaRadioOption {
             checked={checked}
             disabled={disabled}
             onClick={() => this.handleChange()}
+            tabIndex={-1}
```

Comment:
I'm not super clear about how we manage the tab indexes for va-radio and va-radio-option.

There are a few different functions in `va-radio` that seems to be managing tabindex. I think it's `getSlottedNodes` that is dealing with the va-radio-option. Is there anything we can do with the existing tabindex management in the parent component instead of making all of the `input` elements in va-radio-option set to tab-index="-1"? 

https://github.com/department-of-veterans-affairs/component-library/blob/77ab7cf9553d338b85919bdf2b8ae39c11c8d217/packages/web-components/src/components/va-radio/va-radio.tsx#L185-L211

Reply:
Also, how does this work in a page where either no tabindex is set on any element or tabindex is set on some other component? Or are we protected because this is in the Shadow DOM? Do we know that this is the root cause of the double-tab bug? Or is this a hunch that setting the tabindex might fix it?

Reply:
I'm confident this is the right solution. This updates the `input` inside the `va-radio-option`, while the code in `va-radio.tsx` is updating the `va-radio-option` itself. The code is correctly mimicking the behavior of a vanilla radio button group. You tab into the entire group and use the arrow keys to only select one option, then tab out. This is opposed to most inputs, for example checkbox group, where you tab through each option, able to select them as you go, and need to tab through all before leaving the inputs.

The updating tabindex in this code isn't affecting the tab presses while using the components, it's setting the tabindex so it's correct when you tab into it again.

The bug is because of the way `va-radio` handles the focus. The `va-radio-option` holds the focus, not the `input` itself, so when tab is pressed to leave the radio button group, it finds the `input` in the `va-radio-option` and focuses that, making it seem like nothing is changing in focus and needing the second tab press to leave. 

And yes, the tabindex is encapsulated by the shadow DOM, so this code setting the `va-radio-options` tabindex won't affect tab order on the entire page.